tests/integration/TestSuite.php

```diff
@@ -1,4 +1,5 @@
 <?php
```

Comment:
Yea, I did replace it with PHPUnit in the rewrite, but surprisingly there are quite a few users / customers using Riak 1.x and I need to continue to support the legacy client. :(